src/metabase/util.cljc

```diff
@@ -1019,3 +1019,9 @@
      (let [buf (js/Uint8Array. max-length-bytes)
            result (.encodeInto (js/TextEncoder.) s buf)] ;; JS obj {read: chars_converted, write: bytes_written}
        (subs s 0 (.-read result)))))
+
+(def ^:dynamic *ab-value*
```

Comment:
why don't we keep this in `dev.perf.ab-measurement` ?

Reply:
I was trying to put it somewhere accessible and that wouldn't cause circular requires. But putting it somewhere in `dev/` would make the linter reject code that accidentally left in a reference to this var.

Makes sense, I'll move it to `dev.perf.ab-measurement` unless that causes nasty circular references, in which case I'll put it in a sibling namespace.